web/src/lib/domain-logic/getNavBarBusinessTitle.test.ts

```diff
@@ -141,6 +141,38 @@ describe("getNavBarBusinessTitle", () => {
         });
       });
     });
+
+    describe("Nexus DBA", () => {
+      it("shows unamed dba when the user has an unavailable name", () => {
+        const business = generateBusiness({
+          profileData: generateProfileData({
+            businessPersona: "FOREIGN",
+            foreignBusinessType: "NEXUS",
+            businessName: "test business Name",
+            needsNexusDbaName: true,
+            nexusDbaName: "",
+          }),
+        });
+        const navBarBusinessTitle = getNavBarBusinessTitle(business, IsAuthenticated.TRUE);
+        expect(navBarBusinessTitle).toEqual(`${Config.navigationDefaults.navBarUnnamedDbaBusinessText}`);
```

Comment:
[pebble] `` `${...}\` `` is redundant, you can just say the thing. It's already a string